pocs/tests/test_camera.py

```diff
@@ -68,7 +68,7 @@ def camera(request, images_dir):
 
 
 @pytest.fixture(scope='module')
-def counter():
+def counter(camera):
```

Comment:
Does this make any functional changes or fixes?

Reply:
I did this so that there are separate counters for each `camera`. This doesn't have an effect in `develop` right now as (usually) the tests only run with the simulated camera, but it is necessary for the tests to run in `pyro-cameras` (where both simulated and simulated distributed cameras are tested) and would also be required if you were running the tests with real SBIG or FLI cameras attached and configured.